e2e-tests/gef/cypress/integration/upload-files-to-azure.spec.js

```diff
@@ -0,0 +1,157 @@
+import relative from './relativeURL';
+import CREDENTIALS from '../fixtures/credentials.json';
+import applicationDetails from './pages/application-details';
+import automaticCover from './pages/automatic-cover';
+import uploadFiles from './pages/upload-files';
+import manualInclusion from './pages/manual-inclusion-questionnaire';
+
+context('Upload files to Azure', () => {
+  let applicationId;
+  before(() => {
+    cy.reinsertMocks();
+    cy.apiLogin(CREDENTIALS.MAKER).then((token) => token).then((token) => {
+      cy.apiFetchAllApplications(token);
+    }).then(({ body }) => {
+      applicationId = body.items[2]._id;
+      cy.login(CREDENTIALS.MAKER);
+    });
+  });
+
+  describe('Upload files as a Maker', () => {
+    beforeEach(() => {
+      Cypress.Cookies.preserveOnce('connect.sid');
+      cy.visit(relative(`/gef/application-details/${applicationId}`));
+    });
+
+    it('should mark all Eligibility criteria answers as `False`', () => {
+      // Make the deal a Manual Inclusion Application
+      applicationDetails.automaticCoverDetailsLink().click();
+      automaticCover.automaticCoverTerm().each(($el) => {
+        $el.find('[data-cy="automatic-cover-false"]').trigger('click');
+      });
+      automaticCover.continueButton().click();
+      manualInclusion.continueButton().click();
+
+      cy.uploadFile('file1.png', `${manualInclusion.url(applicationId)}/upload`);
+      manualInclusion.uploadSuccess('file1.png');
+      cy.url().should('eq', relative(`/gef/application-details/${applicationId}/supporting-information/manual-inclusion-questionnaire`));
```

Comment:
looks like this test doesn't actuall check for the "false" display, should it? (looking at the `it('...` description)

Reply:
this should be sorted now